specification/elasticsan/resource-manager/Microsoft.ElasticSan/stable/2023-01-01/elasticsan.json

```diff
@@ -2205,6 +2662,86 @@
         }
       },
       "description": "List of private endpoint connections associated with SAN"
+    },
+    "Snapshot": {
+      "type": "object",
+      "required": [
+        "properties"
+      ],
+      "description": "Response for Volume Snapshot request.",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Properties of Volume Snapshot.",
+          "$ref": "#/definitions/SnapshotProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ProxyResource"
+        }
+      ]
+    },
+    "SnapshotList": {
+      "type": "object",
+      "description": "List of Snapshots",
+      "additionalProperties": false,
+      "properties": {
+        "value": {
+          "description": "An array of Snapshot objects.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Snapshot"
+          }
+        },
+        "nextLink": {
+          "description": "URI to fetch the next section of the paginated response.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "SnapshotProperties": {
+      "type": "object",
+      "required": [
+        "creationData"
+      ],
+      "description": "Properties for Snapshot.",
+      "properties": {
+        "creationData": {
+          "description": "Data used when creating a volume snapshot.",
+          "$ref": "#/definitions/SnapshotCreationData"
+        },
+        "provisioningState": {
+          "description": "State of the operation on the resource.",
+          "$ref": "#/definitions/ProvisioningState",
+          "readOnly": true
+        },
+        "sourceVolumeSizeGiB": {
+          "description": "Size of Source Volume",
+          "type": "integer",
+          "format": "int64",
+          "readOnly": true
+        },
+        "volumeName": {
+          "description": "Source Volume Name of a snapshot",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "SnapshotCreationData": {
+      "type": "object",
+      "required": [
+        "sourceId"
+      ],
+      "description": "Data used when creating a volume snapshot.",
+      "properties": {
+        "sourceId": {
+          "type": "string",
+          "description": "If createOption is Copy, this is the ARM id of the source snapshot or disk. If createOption is Restore, this is the ARM-like id of the source disk restore point."
```

Comment:
What's the "createOption"? Not see a parameter with this name for create snapshot.
And if this should be a volume resource Id in same VG (in the first release), please clear specify it. (You can refer the create volume parameter. https://github.com/Azure/azure-rest-api-specs/blob/fcf5d71924afb85df0e73878f6e6012751c4abab/specification/elasticsan/resource-manager/Microsoft.ElasticSan/stable/2023-01-01/elasticsan.json#L2383)

Reply:
Updated

Reply:
Not see the update till now.

Reply:
see it now.